content/news/XXXX-XX-XX-dvs-internals-pt1.md

```diff
@@ -0,0 +1,156 @@
+title: "How Does Timecode Vinyl Actually Work? (Pt. 1)"
+authors: Jan Holthuis
+tags: 2.3, timecode, dvs, vinyl control, serato, cd
+status: draft
+# Analog/Digital diagram via matplotlib: https://gist.github.com/Holzhaus/46ab8ea32d3b89025a0a640db171a14b
+
+
+Since the dawn of DJing, spinning vinyl records has never really gone out of fashion.
+Even when CDs became popular in 90s and most music listeners happily phased out their record players, the classical "two turntables and a mixer" setup stayed the epitome of DJing.
+In recent years, digital DJ controllers became more popular, but there are still many DJs that stick with their Technics SL-1210s turntables.
+
+There's a reason for that: the tactile feeling of manipulating music with your hands is something that controllers have had trouble archieving.
+Motorized controllers like the Rane One try to provide a similar workflow, but it's still to be seen if they will have a notable impact on the prevalence of vinyl.
+
+On the other hand, going digital has its benefits:
+Jumping to different positions instantly inside a track using hotcues - without picking up, moving and dropping the needle - just isn't possible with vinyl records, and carrying around heavy crates of records isn't the best you can do for your back.
+
+Digital Vinyl Systems (DVS) aim to provide the best of both worlds and combine the flexibility and power of digital DJing with the tactile control of traditional vinyl DJing.
```

Comment:
How about also mention also the issue that compared to the 80th only a view tracks are released on vinyls. A pure vinyl DJ has the issue of the niche marked and (without a DVS) no seamless integration with main stream tracks.